Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cargo-chef"
-version = "0.1.22-alpha.0"
```

Comment:
We are not cutting a release for this PR (given that it does not bring functional changes to the binary).

Reply:
Agreed! I left the version change by mistake while testing the jobs... Fixing now!